bin/pipe0-bundler.sh

```diff
@@ -2,9 +2,11 @@
 export BUNDLER_OUTBOX_PATH=${BUNDLER_OUTBOX_PATH:="/mnt/lfss/jade-lta/bundler_stage"}
 export BUNDLER_WORKBOX_PATH=${BUNDLER_WORKBOX_PATH:="/mnt/lfss/jade-lta/bundler_work"}
 export DEST_SITE=${DEST_SITE:="NERSC"}
+export FILE_CATALOG_REST_TOKEN=${FILE_CATALOG_REST_TOKEN:="$(<service-token)"}
+export FILE_CATALOG_REST_URL=${FILE_CATALOG_REST_URL:="https://file-catalog.icecube.wisc.edu"}
 export COMPONENT_NAME=${COMPONENT_NAME:="$(hostname)-pipe0-bundler"}
```

Comment:
This isn't strictly necessary, but I'd move this `COMPONENT_NAME` above `DEST_SITE` in order to keep them in alphabetical order.